requirements.txt

```diff
@@ -62,38 +62,39 @@ oscrypto==1.3.0
 packaging==21.3
 parse==1.19.0
 phonenumbers==8.12.12
+pkgutil_resolve_name==1.3.10
```

Comment:
I can't find why this was pulled in. Any ideas?

Reply:
I actually have no idea. I should probably take it out if it's not being used.